packages/odyssey-react/src/components/TextInput/TextInput.tsx

```diff
@@ -68,14 +69,14 @@ interface CommonProps
   placeholder?: string;
 
   /**
-   * An optional string prefix displayed before the input
+   * An optional prefix displayed before the input
    */
-  prefix?: string;
+  prefix?: string | ReactNode;
```

Comment:
ReactNode includes string!
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1349b640d4d07f40aa7c1c6931f18e3fbf667f3a/types/react/index.d.ts#L232-L241

Reply:
yeah, i know, i thought we were just trying to be explicit here....